src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+}
+
+.field-big {
+  margin: 20px 0;
+}
+
+.input {
+  border: none;
+  height: 68px;
+  width: 100%;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+  padding: 0 0 0 62px;
+}
+
+.input::placeholder {
+  font-family: Avenir, Arial, sans-serif;
```

move it to `html` tag selector
```suggestion
  font-family: inherit;
```

---

Fix it below also

---

https://github.com/mate-academy/layout_search-bar-airbnb/commit/a5242c596481d80f7e78578138cbdd8124cb230b